acceptance-tests/src/test/scala/ch/renku/acceptancetests/pages/NewProjectPage.scala

```diff
@@ -46,10 +47,19 @@ case object NewProjectPage
     visibilityField.click() sleep (1 second)
     visibilityOption(project.visibility).click() sleep (1 second)
 
+    scrollDown
```

Comment:
do we need `scrollDown` now? It seems that we don't.

Reply:
I think it could be necessary if there are many templates, and it does not hurt, so I would leave it.

Reply:
good point